modules/virt-assign-vgpu-passthrough-to-vm.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc
+
+[id="virt-assign-vgpu-passthrough-to-vm_{context}"]
+= Assigning vGPU passthrough to virtual machines
+
+Use the {product-title} web console to assign vGPU passthrough for PCI host devices or mediated hardware devices that your virtual machines uses.
+
+.Prerequisites
+
+* Ensure your cluster and virtual machines are deployed in a bare metal environment. At this time, no other environments are supported.
+
+.Procedure
+
+. Assign the GPU or host hardware devices that you want your cluster and virtual machines to access.
+.. In the {product-title} web console, click *Virtualization > Virtual Machines* from the side menu.
+.. Click the *Details* tab. The *Hardware Devices" field includes links to add or remove *GPU devices* and *Host devices*. Use the minus icon to remove an existing hardware device. You can only add or remove devices from your virtual machine when it is stopped.
```

Comment:
before clicking on the `Details` tab, we need to select the wanted VM from the list by clicking on it's name, I think we should add that as well for clarity.
also the part about add or remove should come only after they opened the editing modal by clicking on the pencil icon in the next paragraph.

Reply:
Maybe we can just add to the first step of the Procedure section something like: "In the OpenShift Container Platform web console, click Virtualization > Virtual Machines from the side menu, and select the VM you want to assign the device to".

Reply:
Missing " in line 18 before the **Hardware Devices**"

Reply:
Great, thanks. Changed to:

_. Assign the GPU or host hardware devices that you want your cluster and virtual machines to access.
.. In the {product-title} web console, click *Virtualization > Virtual Machines* from the side menu.
.. Select the virtual machine to which you want to assign the device.
.. Click the *Details* tab. The *Hardware Devices* field includes links to add or remove *GPU devices* and *Host devices*. Use the minus icon to remove an existing hardware device. You can only add or remove devices from your virtual machine when it is stopped._

Reply:
+1

Reply:
+1